Marlin/temperature.h

```diff
@@ -50,6 +50,10 @@
   #define EXTRUDER_IDX  active_extruder
 #endif
 
+#if ENABLED(PIDTEMP)
+extern int pid_method;
+#endif
+
```

Comment:
Having this variable in 'temperature.h' - gives other .cpp files a hint about `pid_method` when temperature.h is included.
But when `pid_method` is declared 'static' there it is not accessible.
Up to now there is no need for access to `pid_method` from other parts of Marlin - so you could remove this entry here.
As soon as you make `pid_method` configurable in the lcd-menu you need it here - but then the 'static' above does not make sense.